cli/parse_args.ts

```diff
@@ -567,7 +568,7 @@ export function parseArgs<
       o = get(o, key) as NestedMapping;
     });
 
-    const key = keys[keys.length - 1];
+    const key = keys.at(-1)!;
```

Comment:
`keys` is the result of `name.split(".");` and that should never be empty, so can assume non-null